pandas/core/indexing.py

```diff
@@ -944,6 +944,12 @@ def _handle_lowerdim_multi_index_axis0(self, tup):
         except TypeError:
             # slices are unhashable
             pass
+        except KeyError as ek:
+            # raise KeyError if number of indexers match
+            # else IndexingError will be raised
+            if (len(tup) == len(self.obj.index._levels)
```

Comment:
I was wondering should i keep this as it is?
Or should i change `len(self.obj.index._levels)` to `self.obj.index.nlevels`.

Reply:
yes pls change this